Gemfile

```diff
@@ -1,5 +1,4 @@
 source 'https://rubygems.org'
-ruby '2.0.0'
```

Comment:
Yes, I think we should have a specific Ruby version. It's hard to guarantee compatibility with different versions and I don't see that as an important feature.